### ISSUE TITLE
Gum animation window text should not be distorted

### ISSUE THREAD

**Issue by profexorgeek** [vchelaru/Gum#61]
I never saw this in Gum until just recently and I'm wondering if stuff you added from Glue made it start happening. This also happens in the animation editor and in Glue but it never happened in Gum until this morning:

![animation-text-distortion](https://user-images.githubusercontent.com/711100/99911696-6c240480-2cb3-11eb-99eb-c374251b05a5.gif)

**Comment by profexorgeek** [comment]
Okay, Matt and I did some investigating and I found this thread:
https://stackoverflow.com/questions/13448674/wpf-richtextbox-selected-text-distorted

So I tested and this code in Program.cs fixes:
```
static void Main()
{
    System.Windows.Media.RenderOptions.ProcessRenderMode = System.Windows.Interop.RenderMode.SoftwareOnly;
    Application.EnableVisualStyles();
    Application.SetCompatibleTextRenderingDefault(false);
    Application.Run(new MainWindow());
}
```

I would just commit it but I feel like it needs some discussion first. Something with hardware rendering is causing this distortion and it's happening in multiple FRBDK programs. Might be worth investigating more before this band-aid fix?
